scenario_gym/metrics/trajectory.py

```diff
@@ -1,10 +1,13 @@
 import numpy as np
+from shapely.geometry import Polygon
 
+from scenario_gym.entity.pedestrian import Pedestrian
 from scenario_gym.state import State
 
 from .base import Metric
 
 
+# ---------- Speed/Velocity Metrics ----------#
```

Comment:
I would move these in different files rather than have everything in Trajectory. e.g. velocity.pya dn then once for distance.py